xcm/src/v0/order.rs

```diff
@@ -39,33 +39,34 @@ pub enum Order<Call> {
 	///
 	/// Errors:
 	#[codec(index = 1)]
-	DepositAsset { assets: Vec<MultiAsset>, dest: MultiLocation },
+	// TODO: https://github.com/paritytech/polkadot/issues/3547 introduce `, max_assets: u32`
+	DepositAsset { assets: MultiAssetFilter, dest: MultiLocation },
```

Comment:
```suggestion
	DepositAsset { assets: MultiAssetFilter, beneficiary: MultiLocation },
```
Would (something like) this make sense to avoid leading people to think it might trigger another XCM?

Reply:
perhaps, but maybe in a different PR.